_playbooks/playbook-pam.md

```diff
@@ -0,0 +1,367 @@
+
+### Zero Trust Alignment
+NIST Special Publication 800-207 outlines a Zero Trust Architecture that move defenses from static, network-based perimeters to focus on users, assets, and resources. It shifts the operating model toward using telemetry to determine trust rather than location (e.g. if a user is on an agency netowrk and on a government furnished device, they are trusted). WIthin the context of this playbook, step 4 provides recommendations on how to align privileged program technical measures with a zero trust architecture from five elements.
+
+1. User - The user includes both the privileged user and the privileged account. User characteristics used an an access decision may include identity assurance level, credential assurance level, and entitlements.
+2. Device - The device can be assigned to a user or agnostic (a server, password vault, network device, etc.). Device characteristics used in an access decision may include operating system version, device type (government, personal, etc.), device format (mobile, tablet, laptop, etc.), agency compliance state, or location.  
+3. Network - The network is agnostic to the method of accessing the data. This may include public internet or agency intranet. Network characteristics used in an access decision may include network type (public or private), network speed, and other indicators.
+4. Application - The application is the method of serving the data. This can include a front-end application, a command line interface, a database, or other method. This layer may include the user of a per application policy enforcement point or a centralized policy authority point using user, device, and network characteristics to make a one-time and ongoing access decision.
+5. Data - The data includes controls on data sharing, data classification and tagging, preventing data loss, and potentially applying data retention policy. 
+
+[![The five zero trust elements include user, device, network, application, and data]({{site.baseurl}}/assets/playbooks/pam-zt.png)]({{site.baseurl}}/assets/playbooks/pam-zt.png){:target="_blank"}{:rel="noopener noreferrer"}
+
+## Step 2. Define and Identify
+An agency is responsible for managing the privileges of all users. To execute assigned duties, a subset of an agency’s user population may be granted elevated access. The individuals entrusted with elevated access constitute an agency’s privileged user population.
+
+### Define a Privileged User
+An agency’s privileged user population can include a range of accounts with elevated access. Given the broad-reaching nature of an organization’s privileged user population, it is important that an agency understand the user roles, groups, and accounts that constitute its privileged user population. Some common examples include:
+- domain administrator
```

Comment:
Include Linux/Unix root, Oracle SYS, Cisco Enable

Reply:
Also include Windows service accounts, hard-coded SSH keys, embedded credentials in code.

Reply:
Emergency or break-glass accounts.